### ISSUE TITLE
revancedextended.com falsely blocked by badware risks

### ISSUE THREAD

**Issue by K4sum1** [uBlockOrigin/uAssets#18857]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

The website is for an open source YouTube modification. If there was any malware it would be known and taken off GitHub.

### URL(s) where the issue occurs.

'revancedextended.com'

### uBO version

1.50.0

### Browser name and version

Waterfox G5.1.8

### Settings

Not sure what defaults are, and too lazy to check, but I have these ticked

Hide placeholders of blocked elements
Show the number of blocked requests on the icon
Make use of context menu where appropriate

Disable pre-fetching (to prevent any connection for blocked network requests) 
Disable hyperlink auditing 
Block CSP reports 
Uncloak canonical names 

I am an advanced user

**Comment by K4sum1** [comment]
Anyone would rather you download their project over a fork or similar. It's like blocking the yt-dlp website (if it had one) because youtube-dl devs would rather you use their original project instead. Also this is all on GitHub, so idk how they'd get malware in without getting caught like near instantly.

**Comment by stephenhawk8054** [comment]
I think we should only block when there are reports of malwares. We currently don't have descriptions of why the page is blocked, so users just know that this is blocked by "Badware risks" list (I don't think many people can understand uBO syntax to actually know what the filter means).

Also technically the page specifically says it's for a fork (with many of their own developing as far as I see), not the main project, so our argument of referencing main project's discussion would fall short, unless there are words from that fork's devs themselves.

**Comment by erik9012** [comment]
revancedextended.com is not owned by the extended developee

**Comment by inotia00** [comment]
it's not my website
it's a fake site

**Comment by stephenhawk8054** [comment]
> revancedextended.com is not owned by the extended developee

Yeah, that's what I meant by `unless there are words from that fork's devs themselves.`
